src/kayenta/edit/configList.tsx

```diff
@@ -23,18 +23,16 @@ function ConfigList({ configs, selectedConfigId, application }: IConfigListState
     <section className="config-list">
       <ul className="tabs-vertical list-unstyled" style={{ wordBreak: 'break-all' }}>
         {configs.map(config => (
-          <li key={config.id} className={config.id === selectedConfigId ? 'selected' : ''}>
-            <UISrefActive class="active">
-              <UISref to=".configDetail" params={{ id: config.id, new: false, copy: false }}>
-                <a className="heading-4">{config.name}</a>
-              </UISref>
-            </UISrefActive>
-            <div className="body-small color-text-caption caption" style={{ marginTop: '5px', marginBottom: '0' }}>
-              Edited: <FormattedDate dateIso={config.updatedTimestampIso} />
-              <br />
-              <OwnedBy owningApplications={config.applications} currentApplication={application} />
-            </div>
-          </li>
+          <UISref key={config.id} to=".configDetail" params={{ id: config.id, new: false, copy: false }}>
```

Comment:
yeah — the `tab-vertical` thing in the styleguide doesn't play nice with that structure (need to move the padding from the `<li>` to the `<a>`), so I originally opted to omit it. I just went in and did some overrides, but we should likely revisit that tab style to support better handling of anchors rather than forcing tiny click targets and/or overrides in app code.

Reply:
(5270374)